test/accelerators/NNPA/backend/CMakeLists.txt

```diff
@@ -187,7 +187,7 @@ set(NNPA_TEST_LIST
 
     # ==OP== GRU
     # ==MIN== 7
-    # ==LIM== - `direction` and `hidden_size` in `W` must have static dimensions.<br>- `R` must have static dimensions.<br>- If `B` and `initial_h` are given, they must have static dimensions.<br>- `sequence_lens` is not supported.<br>- `activations` must be `["Sigmoid", "Tanh", "Tanh"]`.<br>- `clip` is not supported.<br>- `linear_before_reset` must be 1.<br>- `layout` is not supported.
+    # ==LIM== - `direction` and `hidden_size` in `W` must have static dimensions.<br>- `R` must have static dimensions.<br>- If `B` and `initial_h` are given, they must have static dimensions.<br>- `sequence_lens` is not supported for bidirectional GRU.<br>- `activations` must be `["Sigmoid", "Tanh", "Tanh"]`.<br>- `clip` is not supported.<br>- `linear_before_reset` must be 1.<br>- `layout` is not supported.
```

Comment:
@chentong319 When updating the NNPA doc.... can you update this file instead `test/accelerators/NNPA/backend/CMakeLists.txt` and then issue this command `python3 documentOps.py --arch NNPA --input ../test/accelerators/NNPA/backend/CMakeLists.txt --path . --notes` to update `docs/SupportedONNXOps-NNPA.md` instead of directly touching the file. Since it is auto generated :)